parcel_test.go

```diff
@@ -30,58 +32,139 @@ func getTestParcel() Parcel {
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	db, errDB := sql.Open("sqlite", "tracker.db")
+
+	require.Nil(t, errDB)
+	require.NotNil(t, db)
+
+	defer func() {
+		err := db.Close()
+		assert.Nil(t, err)
+	}()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
+	id, errAdd := store.Add(parcel)
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	require.Nil(t, errAdd)
+	require.NotEmpty(t, id)
 
 	// get
+	p, errGet := store.Get(id)
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
+	require.Nil(t, errGet)
+
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	// это невозможно т к number получаем во время записи в базу данных
+	wont := reflect.ValueOf(parcel)
+	get := reflect.ValueOf(p)
+
+	// protects us from out of range in cycle
+	require.Equal(t, wont.Type(), get.Type())
+
+	lenFields := wont.NumField()
+
+	// skip number -> i:=1
+	for i := 1; i < lenFields; i++ {
+		assert.Equal(
+			t,
+			wont.Field(i).Interface(),
+			get.Field(i).Interface(),
+		)
+	}
 
 	// delete
+	errDel := store.Delete(id)
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
+	require.Nil(t, errDel)
+
 	// проверьте, что посылку больше нельзя получить из БД
+	_, errCheck := store.Get(id)
+	require.NotNil(t, errCheck)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```